### ISSUE TITLE
Holder signature in presentation does not indicate consent

### ISSUE THREAD

**Issue by mavarley** [w3c/vc-data-model#472]
3.4 Example 2:
```
// digital signature by Pat on the presentation establishes consent and
// protects against replay attacks
```
I don’t see how this represents consent. There is a missing piece about the wallet that collected the consent. Further, can this be extended to include the termsOfUse (described in the picture) and link to an “evidence” field which could be a consent Receipt? 

(I don’t think we want to make consent an explicit part of the presentation spec, but  but I’d like to ability to add it when necessary...).

**Comment by stonematt** [comment]
Anything in this document other than the data model and syntactic representations is out of scope. The group is expressly prevented from any other definitive guidance.

This comment represents something about intent of the users and.  These issues are in the domain of protocol and user agents processing and are out of scope.  We will update the comment and remove that assertion once we're in CR.

**Comment by mavarley** [comment]
works for me, thank-you.

**Comment by burnburn** [comment]
In call today @stonematt agreed to write a PR.

**Comment by msporny** [comment]
The PR has multiple positive reviews and is now merged. The next step is to ask the WG to make sure we have closure on the issue for the purposes of CR exit.

**Comment by stonematt** [comment]
Resolved on VCWG call 4/16/2019.
RESOLVED: Issue #472 is addressed by PR #476, which makes non-normative changes noting that consent isn't implied by signatures on presentations as requested by the reviewer. The PR has been approved by multiple parties and has been merged. Issue #472 should be closed.

Plan to close in 7days
